addons/mail/static/src/models/message/message.js

```diff
@@ -612,6 +639,20 @@ function factory(dependencies) {
                 'originThreadIsModeratedByCurrentPartner',
             ],
         }),
+        /**
+         * States whether `threadname` and `subject` contain similar
+         * values except it's prefix.
```

Comment:
I would end this phrase after "values". And make a new phrase to explain if the subject contains extra prefix but the rest is the same it is considered similar to thread name too.